### ISSUE TITLE
Hide tag graph from tag pages like /tag/ui with a toggle button

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#4646]
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://code.publiclab.org/#r=all), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @publiclab/reviewers to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

![image](https://user-images.githubusercontent.com/24359/51340607-5f05b100-1a5d-11e9-83ef-47097864cbc5.png)

We're looking to simplify the tag pages as part of the ongoing UI projects, and the tag graphs will be moved to their own page soon (#4140).

As a first steps, we're going to hide them and make them display only when you click a "graph" button. That button can later be the link button to the separate page. 

The code can be found here: 

https://github.com/publiclab/plots2/blob/9f7cfc47ded41ed3a7370c66a9c4f7af9704af59/app/views/tag/show/_user_controls.html.erb#L4

Just after this line, we can add a similar button with a "graph" icon:

```htlm
  <a onClick="$('.tag-graph').toggle();" class="btn btn-default btn-sm"><i class="fa fa-bar-chart"></i> Stats</a>
```

See how we've made it toggle any element with the CSS class `tag-graph`? So let's wrap the graph area in an element with that div:

https://github.com/publiclab/plots2/blob/9f7cfc47ded41ed3a7370c66a9c4f7af9704af59/app/views/tag/show.html.erb#L7-L9

Let's make that look like:

```
  <% if @tags.first && params[:id][-1..-1] != "*" %>
    <div class="tag-graph" style="display:none;">
      <%= render partial: "tag/graph", locals: { tags: @tags } %>
    </div> 
 <% end %>
```

That'll hide it by default, but enable it to be shown with the "graph" button. 

We'd love help with this, or with turning it into a first-timers-only issue!

**Comment by IgorWilbert** [comment]
Hello @jywarren ! May I work on this? If I don't finish it in a few days, I'll try to add more details to turn it into a FTO.

**Comment by jywarren** [comment]
that would be great, thank you!!!

**Comment by IgorWilbert** [comment]
@jywarren I'm sorry, I didn't work on this until now because the setup for PublicLab.Editor seems to be more simple than the one for plots2 and the editor needs some love of new contributors. Please feel free to assign this issue to someone else.

**Comment by gauravano** [comment]
Hey @IgorWilbert, no issue! Although, thanks for trying and feel free to ask your installation doubts in chatroom or in the issue whose link I provided you. Thanks!

**Comment by chirag-singhal** [comment]
@IgorWilbert hy!  Can i solve this issue, I am a first time contributor?

**Comment by cesswairimu** [comment]
@chirag-singhal I don't know if you are still working on this but I somehow solved this issue on #4140  . Please let me know if you need help finding another issue to work on. 
Closing this, 
sorry

**Comment by gauravano** [comment]
Hey @chirag-singhal, you can look for new issues at https://code.publiclab.org/# r=all and if you can't see any unclaimed, feel free to comment here, we will open a issue for you as @cesswairimu mentioned. Thanks!
